src/models/config.ts

```diff
@@ -1,19 +1,12 @@
-import { PresenceStatusData } from 'discord.js';
+import { PresenceData } from 'discord.js';
 
 export interface Config {
   discord: {
     richpresence: {
       interval: number,
       statusType: number,
-      states: [
-        {
-          status: PresenceStatusData | string,
-          activity: {
-            name: string,
-            type: number
-          }
-        }
-      ]
+      states: PresenceData[]
```

Comment:
consider removing states and just an array

Reply:
interval is included in presence